### PR TITLE
Verify signature of source container image

### PR DESCRIPTION
**Pull request by lcarva**
Ref: EC-265

### CODE REVIEW

File: policy/beta/source_image.rego [hunk 1]
```diff
@@ -42,6 +42,27 @@ _source_image_errors contains error if {
...
+deny contains result if {
+	some error in _source_image_sig_errors
+	result := lib.result_helper(rego.metadata.chain(), [error])
+}
+
+_source_image_sig_errors contains error if {
+	some img in _source_images
+	info := ec.sigstore.verify_image(img, lib.sigstore_opts)
+	some raw_error in info.errors
+	error := sprintf("Image signature verification failed for %s: %s", [img, raw_error])
```
Comment (zregvart) [inline policy/beta/source_image.rego:63]: I wonder why the code coverage check didn't fail for this line, I don't see a test reaching this

**Review by zregvart — COMMENTED** [review]
Perhaps we test coverage for the errors from `ec.sigstore.verify_image`

File: policy/beta/source_image.rego (hunk 1 above)
Comment (lcarva) [reply policy/beta/source_image.rego:63]: test_missing_signature does. See the expected error messages [here](https://github.com/enterprise-contract/ec-policies/pull/946/files#diff-24cbb169deacc3371a03a72775c6e0c70311d24131c4d1e9c93166dbb01c1d99R249-R261).

**Review by zregvart — APPROVED** [review]

File: policy/beta/source_image.rego (hunk 1 above)
Comment (zregvart) [reply policy/beta/source_image.rego:63]: Thanks, missed this somehow...

**Pull request merged**